text/java/0003-clojure-tools.md

```diff
@@ -0,0 +1,33 @@
+# Clojure Tools Buildpack
+
+## Summary
+
+A Clojure Tools buildpack for building Clojure applications using [deps][d] and [tools build][t] has been created [by the community][b].  It is based on the same technologies and is nearly indistinguishable from the other Paketo Buildpacks, especially the JVM build-system buildpacks, and it should be adopted into the project.
+
+[d]: https://clojure.org/guides/deps_and_cli
+[t]: https://clojure.org/guides/tools_build
+[b]: https://github.com/eddumelendez/clojure
+
+## Motivation
+
+This is an additional buildpack, for building JVM-based applications based on Clojure, and offers yet another workload that can be built by Paketo buildpacks.
+
+## Detailed Explanation
+
+N/A
+
+## Rationale and Alternatives
+
+N/A
+
+## Implementation
+
+This buildpack will be adopted by the Java Buildpacks team and transferred to the `paketo` GitHub organization.  The original author will retain full commit rights to the repository and be entitled to a full say in the direction and features of the buildpack.
```

Comment:
Would you prefer to be added as a contributor? We had this discussion previously, but with the number of buildpacks you've contributed it probably makes sense to just add you as a contributor to the team. Thoughts?

Reply:
Cool. This change looks good. We'll have to follow governance and call for a vote to add you as a contributor, but I don't think there should be a problem with that. Thanks again for all your contributions!